configs/dev-config.yaml

```diff
@@ -130,3 +130,152 @@ scene_data:
     "4ec9599fc203d176a301536c2e091a19bc852759b255bd6818810a42c5fed14a": 18
     "9400f1b21cb527d7fa3d3eabba93557a18ebe7a2ca4e471cfe5e4c5b4ca7f767": 19
     "f5ca38f748a1d6eaf726b8a42fb575c3c71f1864a8143301782de13da2d9202b": 20
+  camera_properties:
+    2:
+      position:
+        - -2
+        - 1.3
+        - 5.2
+      look_controls: true
+      wasd_controls: true
+      cursor_properties:
+        shape: crosshairs
+        asset_id: 7
+        position:
+          - 0
+          - -1
+          - -1
+    5:
+      position:
+        - 0
+        - 3
+        - 3
+      look_controls: true
+      wasd_controls: true
+      cursor_properties:
+        shape: crosshairs
+        asset_id: 1
+        position:
+          - 0
+          - 0
+          - 0
+    7:
+      position:
+        - -2
+        - 1.3
+        - 5.2
+      look_controls: true
+      wasd_controls: true
+      cursor_properties:
+        shape: crosshairs
+        asset_id: 7
+        position:
+          - 0
+          - -1
+          - -1
+    30:
+      position:
+        - 3.809
+        - 3.5
+        - 2.993
+      look_controls: true
+      wasd_controls: true
+      cursor_properties:
+        shape: crosshairs
+        asset_id: 7
+        position:
+          - 0
+          - -1
+          - -1
+    43:
+      position:
+        - -1.628
+        - 3.5
+        - 10.027
+      look_controls: true
+      wasd_controls: true
+      cursor_properties:
+        shape: crosshairs
+        asset_id: 7
+        position:
+          - 0
+          - -1
+          - -1
+  position:
+    2:
+      - 0.0
+      - 0.03
+      - 0.0
+    5:
+      - 0.0
+      - 0.13
+      - 0.0
+    7:
+      - 0.0
+      - 0.03
+      - 0.0
+    13:
+      - 0.0
+      - 0.2
+      - 10.0
+    30:
+      - 0.0
+      - 0.1
+      - 0.0
+    43:
+      - 0.0
+      - 0.1
+      - 0.0
+  rotation:
+    2:
+      - 0.0
+      - 180.0
+      - 0.0
+    5:
+      - 0.0
+      - 0.0
+      - 0.0
+    7:
+      - 0.0
+      - 180.0
+      - 0.0
+    13:
+      - 0.0
+      - 0.0
+      - 0.0
+    30:
+      - 0.0
+      - 90.0
+      - 0.0
+    43:
+      - 0.0
+      - 90.0
+      - 0.0
+  scale:
+    2:
+      - 1.0
+      - 1.0
+      - 1.0
+    5:
+      - 1.0
+      - 1.0
+      - 1.0
+    7:
+      - 1.0
+      - 1.0
+      - 1.0
+    13:
+      - 0.1
+      - 0.1
+      - 0.1
+    30:
+      - 0.425
+      - 0.425
+      - 0.425
+    43:
+      - 0.425
+      - 0.425
+      - 0.425
+  hints:
+    - "You can use your mouse to pan around, and use the arrow keys / WASD to move around"
+    - "If you are seeing this page and aat a ren't the admin, please contact Calgary ConnecTeen to let them know that a solution is missing :)"
```

Comment:
Same here

Reply:
Fixed!

Reply:
I think there is still a typo. `...page and aat aren't the...`. The "aat" shouldn't be there.

Reply:
Updated